template/test-report.vuln-card.hbs

```diff
@@ -26,6 +26,7 @@
                 {{/if_eq}}
             </li>
 
+            {{#if_not_eq list.[0].packageManager "Unmanaged (C/C++)"}}
```

Comment:
Is it safe to access the list directly like this?

Reply:
I guess yes - I saw a lot of usage in the code...